### PR TITLE
Release/3.4.2

### PR DESCRIPTION
**Pull request by dreamwhisper**
See #402

### How to test

1. Code review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Genesis Sample Theme Changelog
 
+## [3.4.2] - 2021-12-19
```
Comment (kienstra) [inline CHANGELOG.md:3]:
```suggestion
## [3.4.2] - 2022-12-19
```

**Review by kienstra — APPROVED** [review]
Hi @dreamwhisper,
This looks good, pending a very minor comment.

File: CHANGELOG.md (hunk 1 above)
Comment (dreamwhisper) [reply CHANGELOG.md:3]: @kienstra 😆 And, it is almost 2023...

Comment (kienstra) [reply CHANGELOG.md]: Haha, once I released a version of GCB with the wrong year

**Pull request merged**